src/components/MainTable/MainTable.scss

```diff
@@ -0,0 +1,4 @@
+caption {
```

Comment:
Why is this style applied to MainTable? This should be a global style accompanied with an issue upstream.

Reply:
Issue raised upstream here: https://github.com/canonical-web-and-design/vanilla-framework/issues/2663

I didn't want to apply it globally as I see this as a local override until it has been agreed upstream in Vanilla?

Reply:
Thanks, could you add a comment of the issue so the styling can be removed when resolved. I'm allergic to custom styling.

Reply:
aiui this will actually be applied globally as there is no `main-table` wrapper style. It's just defined here....which may make it challenging to debug later.

Can you add this wrapper class?